env.sh

```diff
@@ -20,4 +20,7 @@ LOG_FILE="/var/log/azureDiskInspectSvc.log"
 CREDSCANSERVICEVOLUMENAME="CredScanSvc"
 
 # TODO: replace this with envionment specific AppInsights instrumentation key
-APPINSIGHTS_KEY='00000000-0000-0000-00000000000000000'
\ No newline at end of file
+APPINSIGHTS_KEY='00000000-0000-0000-00000000000000000'
+# Replace this with environment specific AppInsights Endpoint. Below is the defualt for Azure Public environemnt.
+# Will be relace by environment variable for other clouds while service deployment
```

Comment:
Instead of "while service deployment", should it be "during service deployment via script"?

Reply:
Done